comments.php

```diff
@@ -22,56 +22,47 @@
 
 <div id="comments" class="comments-area">
 
-	<?php
-	// You can start editing here -- including this comment!
-	if ( have_comments() ) :
-	?>
-		<h2 class="comments-title">
-			<?php
-			$comment_count = get_comments_number();
-			if ( 1 === $comment_count ) {
-				printf(
```

Comment:
@westonruter isn't this doing the same this as `_nx()` below?

Reply:
This is copied from _s: https://github.com/Automattic/_s/blob/388951f1eeda411a066d2eab47d6ae5165743d87/comments.php#L30-L44

Some languages have different forms for singular in particular and that is why there is a separate condition here I believe.